birdhouse/components/jupyterhub/pre-docker-compose-up

```diff
@@ -1,8 +1,7 @@
 #!/bin/sh
 
 # this is external in docker-compose.yml so have to create here
-# no error if already exist, just an error message
-docker network create jupyterhub_network
+docker network create jupyterhub_network || true
```

Comment:
I think a warning log (or maybe debug since "intended behavior") should be reported here. Maybe the jupyter network already existing could be the cause of an underlying issue that would be hard to identify by silencing the "duplicate".

Reply:
I think this just print the error message to console.  Whether with or without `|| true` the message will still be printed anyways so no "silencing" risk here.  Or I might have misunderstood the effect of `|| true`?

Reply:
@tlvu yes that is correct

Reply:
Ok, that is fine then.